### DISCUSSION TITLE
Non Yoga FlexBox implementation to Yoga FlexBox?

### DISCUSSION THREAD

**Discussion by Kewl-Ranch-Dorito** [cashapp/redwood#1308] [General]
Hi! I noticed there was a recent switch from using a Compose FlexBox implementation to switching to Yoga.
https://github.com/cashapp/redwood/pull/1216
https://github.com/cashapp/redwood/pull/1186

I was curious of the reasoning?
It appeared the previous FlexBox implementation found a way to support shrink and grow without having to use MultiMeasureLayout, so it appeared the challenge was solved.
Curious about the decision.

Thank you!

**Comment by colinrtwhite** [comment]
Hi, we moved over to the Yoga implementation of flexbox since it fixed a number of rendering edge cases we were seeing in our internal apps. The Yoga flexbox implementation was built from the ground up with a multiplatform layout system in mind whereas the Google flexbox implementation was built for Android.

> It appeared the previous FlexBox implementation found a way to support shrink and grow without having to use MultiMeasureLayout, so it appeared the challenge was solved.

Where do you see that? It was my understanding the shrink and grow required multiple layout passes in their implementation as well.
